src/components/timeline/timeline-legend.tsx

```diff
@@ -114,14 +112,16 @@ const TimelineLegend = ({ entries }: TimelineLegendProps) => {
     <div css={containerStyles}>
       <div css={timelineLegendStyles}>
         {entries.map((entry, i) => {
+          const titleSlug = slugify(entry.title, { lower: true });
+          const link = `/issues-chronology/${titleSlug}/`;
           const Icon = entry.icon;
           return (
-            <a key={i} href={`#${entry.anchor}`}>
-              <div key={i} css={[entryWrapperStyles, colorCss(entry.color)]}>
+            <Link key={i} to={link} className={titleSlug}>
```

Comment:
since we're relying on these `classNames` and `titleSlugs` aligning, it might be nice to have a jest test (or something else) that checks they do